padl/util_transforms.py

```diff
@@ -108,3 +108,58 @@ class IfTrain(IfInMode):
 
     def __init__(self, if_: Transform, else_: Optional[Transform] = None):
         super().__init__(if_, 'train', else_)
+
+
+class Try(ClassTransform):
+    """ Perform *transform*. If this fails with any exception from *exceptions*, perform
+    *catch_transform*.
```

Comment:
you could also explain the else and finally cases here

Reply:
Right